cmd/vic-machine/main.go

```diff
@@ -72,9 +79,21 @@ func main() {
 	} else {
 		app.Version = fmt.Sprintf("%s-%s", BuildID, CommitID)
 	}
+	// Open log file
+	f, err := os.OpenFile(LOG_FILE, os.O_RDWR|os.O_CREATE|os.O_APPEND, 0644)
+	if err != nil {
+		fmt.Printf("Error opening logfile %s: %v", LOG_FILE, err)
+	}
+	defer f.Close()
+
+	// Initiliaze logger with default TextFormatter
+	log.SetFormatter(&log.TextFormatter{ForceColors: true, FullTimestamp: true})
+	// SetOutput to io.MultiWriter so that we can log to stdout and a file
+	log.SetOutput(io.MultiWriter(os.Stdout, f))
```

Comment:
Let's also avoid writing to 'f' when nil.  Above you can start with:
```go 
logs := []io.Writer{io.Stdout}
```

When os.OpenFile is ok, then:

```go
logs = append(f, logs)
```
And here:

```go
log.SetOutput(io.MultiWriter(logs...))
```